Scripts_to_create_table/entities.sql

```diff
@@ -0,0 +1,9 @@
+CREATE SEQUENCE entities_seq;
+CREATE TABLE entities(
+   entity_id integer PRIMARY KEY default nextval('entities_seq'),
+   name text NOT NULL,
+   pseudonym text[],
```

Comment:
The table 'entities' is only for team2 to get the artist name so that they can scrap more information for that particular artist. The field name acts as a vocabulary for them. This is not the main database that the group 3,4,5,6 will be querying.